### PR TITLE
LWG-3851: `chunk_view::inner-iterator` missing custom `iter_move` and `iter_swap`

### LINKED ISSUE #3432: LWG-3851 `chunk_view::inner-iterator` missing custom `iter_move` and `iter_swap`
LWG-3851 `chunk_view::`*`inner-iterator`* missing custom `iter_move` and `iter_swap`

### PR DESCRIPTION
**Pull request by JMazurkiewicz**
Closes #3432.

### CODE REVIEW

File: tests/std/tests/P2442R1_views_chunk/test.cpp [hunk 1]
```diff
@@ -464,8 +466,9 @@ template <ranges::input_range Rng, class Expected>
 constexpr bool test_input(Rng&& rng, Expected&& expected) {
     using ranges::chunk_view, ranges::equal, ranges::iterator_t, ranges::sentinel_t;
 
-    using V = views::all_t<Rng>;
-    using R = chunk_view<V>;
+    using V  = views::all_t<Rng>;
+    using BI = ranges::iterator_t<V>;
```
Comment (StephanTLavavej) [inline tests/std/tests/P2442R1_views_chunk/test.cpp:470]: There's a using-declaration 2 lines above.

**Review by StephanTLavavej — APPROVED** [review]

**Comment by StephanTLavavej** [comment]
Thanks! I pushed a one-line change because I couldn't bear to live with a redundant qualifier. Please forgive me, CI system, for the unnecessary load. :joy_cat: :zany_face:

**Comment by StephanTLavavej** [comment]
I'm a greedy kitty who wants treats :smirk_cat: and is *speculatively* mirroring this to the MSVC-internal repo - please notify me if any further changes are pushed.

**Review by strega-nil-ms — APPROVED** [review]

**Comment by StephanTLavavej** [comment]
Thanks for the tasty LWG issue resolution! :cat2: :yum: :heart_eyes_cat:

**Pull request merged**